src/metabolic_ninja/schemas.py

```diff
@@ -27,7 +27,34 @@ class Meta:
         strict = True
 
 
+class ProductSchema(Schema):
```

Comment:
It could be useful to have some documentation about where this object is coming from and its data structure